vmp/api/user_client.go

```diff
@@ -71,6 +71,9 @@ func (APIClient *UserAPIClient) GetCustomerUser(accountNumber string, customerUs
 func (APIClient *UserAPIClient) AddCustomerUser(accountNumber string, body *CustomerUser) (int, error) {
 	// TODO: support custom id types, not just Hex ID ANs
 	baseURL := fmt.Sprintf("v2/pcc/customers/users?idtype=an&id=%s", accountNumber)
+	if APIClient.PartnerID == 0 {
```

Comment:
Didn't we remove the requirement for partner_id? It should be implied from PCC tokens and only be required for Uber tokens